services/horizon/internal/actions_transaction.go

```diff
@@ -211,7 +230,7 @@ func (action *TransactionCreateAction) loadResource() {
 	switch err := action.Result.Err.(type) {
 	case *txsub.FailedTransactionError:
 		rcr := horizon.TransactionResultCodes{}
-		resourceadapter.PopulateTransactionResultCodes(action.R.Context(), &rcr, err)
+		_ = resourceadapter.PopulateTransactionResultCodes(action.R.Context(), &rcr, err)
```

Comment:
why is it needed to assign to an used var here?

Reply:
its error handling, the IDE screams when he sees an unhandled error, ( i have nothing to do with that error so I assign it to _